src/components/AddAddressesButton.tsx

```diff
@@ -2,7 +2,6 @@ import { MinusSquareFilled, PlusOutlined } from "@ant-design/icons";
 import { Alert, Button, Form, Input, Modal, Space } from "antd";
 import _ from "lodash";
 import React, { useState } from "react";
-import { validAddressRegex } from "../util/addresses";
```

Comment:
Might as well get rid of the regex, no?

Reply:
[684877a](https://github.com/GridPlus/lattice-manager/pull/228/commits/684877a9f2788380c7414687f278e683c34f8e76)